### PR TITLE
Uyuni BV: Add workaround for Deblike minions

### PR DESCRIPTION
**Pull request by nodeg**
This will

- add a workaround for Debian 10 and Ubuntu 20.04
- remove the Debian 9 comment

### CODE REVIEW

File: terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf [hunk 1]
```diff
@@ -909,6 +925,10 @@ module "debian11-sshminion" {
   }
   use_os_released_updates = false
   ssh_key_path            = "./salt/controller/id_rsa.pub"
+
```
Comment (Bischoff) [inline terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf:928]: no, you do not need it for Debian 11

File: terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf [hunk 2]
```diff
@@ -517,6 +523,10 @@ module "debian11-minion" {
   auto_connect_to_master  = false
   use_os_released_updates = false
   ssh_key_path            = "./salt/controller/id_rsa.pub"
+
```
Comment (Bischoff) [inline terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf:526]: No, you do not need it for Debian 11

File: terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf [hunk 3]
```diff
@@ -881,8 +895,6 @@ module "ubuntu2204-sshminion" {
   ssh_key_path            = "./salt/controller/id_rsa.pub"
 }
 
-// Debian 9 is not supported by 4.3
```
Comment (Bischoff) [inline terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf:884]: ... or you change it to:
```
// Debian 9 is not supported by Uyuni
```

File: terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf [hunk 4]
```diff
@@ -479,8 +483,6 @@ module "ubuntu2204-minion" {
...
```
Comment (Bischoff) [inline terracumber_config/tf_files/Uyuni-Master-build-validation-NUE.tf:482]: ... or you change it to:
```
// Debian 10 is not supported by Uyuni
```

File: terracumber_config/tf_files/Uyuni-Master-build-validation-PRV.tf [hunk 5]
```diff
@@ -713,6 +721,10 @@ module "debian11-minion" {
   auto_connect_to_master  = false
   use_os_released_updates = false
   ssh_key_path            = "./salt/controller/id_rsa.pub"
+
```
Comment (Bischoff) [inline terracumber_config/tf_files/Uyuni-Master-build-validation-PRV.tf:724]: same

File: terracumber_config/tf_files/Uyuni-Master-build-validation-PRV.tf [hunk 6]
```diff
@@ -1166,6 +1186,10 @@ module "debian11-sshminion" {
   }
   use_os_released_updates = false
   ssh_key_path            = "./salt/controller/id_rsa.pub"
+
```
Comment (Bischoff) [inline terracumber_config/tf_files/Uyuni-Master-build-validation-PRV.tf:1189]: same

**Review by Bischoff — CHANGES_REQUESTED** [review]
see remarks

**Review by Bischoff — APPROVED** [review]

**Review by srbarrios — APPROVED** [review]

**Pull request merged**